src/System.CommandLine/ParseResult.cs

```diff
@@ -99,7 +98,7 @@ public class ParseResult
         /// Gets the directives found while parsing command line input.
         /// </summary>
         /// <remarks>If <see cref="CommandLineConfiguration.EnableDirectives"/> is set to <see langword="false"/>, then this collection will be empty.</remarks>
-        public IReadOnlyDictionary<string, IReadOnlyList<string>> Directives { get; }
+        public IReadOnlyDictionary<string, IReadOnlyList<string>> Directives => _directives ??= new ();
```

Comment:
```suggestion
        public IReadOnlyDictionary<string, IReadOnlyList<string>> Directives => _directives ??= new();
```

Reply:
I think that this is a matter of personal preference. We should discuss it with others, choose one policy and enforce it with static code analysis tools.

Reply:
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1000.md